ultra/ultra/scenarios/common/distributions.py

```diff
@@ -146,7 +146,7 @@
             },
             "south-north": None,  # blocking
             "west-east": {
-                "vehicles": 1000,
+                "vehicles": 200,
```

Comment:
Interesting, okay good call. How did you decide on the number 200?

Reply:
The 200 social vehicles is still large amount, however I sort of accounted for some worst case scenarios. It is a bit hard to explain here, but I can reduce the number to 100 even?

Reply:
Oh okay, I see. Even with the changes of #1014, the scenario will not run out of social vehicles?

Reply:
Even with `endless_traffic` turned off, we have a sufficient amount of vehicles to entertain for more 200 secs (2000 timesteps)